crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
```

Comment:
Where is the company insights section? Is this supposed to link somewhere?

Reply:
In RST, when text is \*wrapped in single asterisks\*, it is displayed in *italic*. When that text is \*\*wrapped in double asterixes\*\* is displayed in **bold**.

You can tell if some RST code leads somewhere (external link or internal reference) if it uses one of the formattings shown [here](https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_cheat_sheet.html#hyperlinks).

Reply:
> Where is the company insights section? Is this supposed to link somewhere?

It's a section in the extension. You can see it in one of the screenshots.